vizro-core/docs/pages/cheatsheet/cheatsheet.html

```diff
@@ -732,206 +1031,213 @@ <h2 class="section-title">Page layout & navigation</h2>
-      <div class="main-grid">
-        <!-- Data Management Section -->
-        <section class="section wide-section">
-          <div class="section-header">
-            <h2 class="section-title">
-              <a
-                href="https://vizro.readthedocs.io/en/stable/pages/user-guides/data/"
-                >Data management</a
-              >
-            </h2>
-          </div>
-          <div class="components-grid-data-management">
-            <div class="subsection">
-              <div class="subsection-title">Data loading</div>
-              <div class="subsection-description">
-                Load and manage data from various sources using the centralized
-                data manager.
+        </div>
+        <!-- end of fourth card Page layout-->
+        <!-- beginning of fifth card Data-->
+        <div class="card pink" id="data-management-section">
+          <h2 class="section-title">
+            <span
+              class="section-icon"
+              style="
+                background: linear-gradient(135deg, #f472b6, #db2777);
+                color: white;
+              "
+              >💾</span
+            >
+            Data Management
+          </h2>
+          <div class="three-column-grid">
+            <div class="component-card">
+              <div class="card-title">
+                <a
+                  href="https://vizro.readthedocs.io/en/stable/pages/user-guides/data/#static-data"
+                  >Static data</a
+                >
+              </div>
+              <div class="card-description">
+                Apply the data_frame directly to your chart if your data will
+                not refresh while the dashboard is running.
               </div>
               <div class="code-block-container">
                 <div class="code-block">
-                  <pre><code>import pandas as pd
-from vizro.managers import data_manager
+                  <pre><code class="language-python">import pandas as pd
 
 # Load data
 df = pd.read_csv("data.csv")
-data_manager["my_data"] = df
 
 # Use in components
 vm.Graph(
     figure=px.scatter(
-        data_frame="my_data",
+        data_frame=df,
         x="x_col",
         y="y_col"
     )
 )</code></pre>
                 </div>
-                <button class="copy-button" onclick="copyCode(this)">
-                  Copy
-                </button>
+                <button class="copy-btn" onclick="copyCode(this)">Copy</button>
+              </div>
+              <div class="highlight-box">
+                <div class="highlight-title">
+                  <span class="mdi mdi-lightbulb-on-outline"></span>
+                  <strong>Static data tips:</strong>
+                </div>
+                <div class="highlight-content">
+                  • The same can be achieved by assigning the data_frame to the
+                  data_manager key such as
+                  <code>data_manager["my_data"] = df</code>.
```

Comment:
I added one more sentence to fulfil the explanation. I didn't test the solution 😄  

```suggestion
                  • The same can be achieved by assigning the data_frame to the
                  data_manager key such as
                  <code>data_manager["my_data"] = df</code>.
                  Then, you can reference this key when specifying the data_frame argument in your figure, like:
<code>figure(data_frame="my_data")</code>
```